solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -82,25 +83,55 @@ public void analyzeFiles(SensorContext context, List<InputFile> files) {
     for (InputFile file : files) {
       String lastAnalyzedFile = file.toString();
       if (inSonarQube(context)) {
-        LOG.debug("Analyzing: " + lastAnalyzedFile);
-        getSyntaxHighlighting(context, file).save();
+        try {
+          LOG.debug("Analyzing: " + lastAnalyzedFile);
+          SolidityParser parser = Utils.returnParserUnitFromParsedFile(file.contents());
+          getSyntaxHighlighting(parser, context, file).save();
+          saveFileMeasures(context, computeMeasures(parser, fileLinesContextFactory.createFor(file), file), file);
+          if (inSonarQube(context)) {
+
+          }
+        } catch (IOException e) {
+          e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=16&issues=AWQTIo4yYwZU9vvrXIhe&open=AWQTIo4yYwZU9vvrXIhe)

[ISSUE_KEY]: AWQTIo4yYwZU9vvrXIhe